File: crucial_algo/sorting/bubble_sort.py
def sort(arr: list) -> list:
    n = len(arr)
    

    for i in range(n-1):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                
                arr[j], arr[j+1] = arr[j+1], arr[j]
                
    return arr

arr = [5,2,1,1,7,6,6]
print(sort(arr))

File: crucial_algo/sorting/bubbleSort.go
package main

import "fmt"


func b_sort(arr []int) []int {
	n := len(arr)
	for i := 0; i < n - 1; i++ {
		for j := 0; j < n - 1 - i; j++ {
			if arr[j] > arr[j+1] {
				arr[j], arr[j+1] = arr[j+1], arr[j]
			}
		}
	}
	return arr
}

func main(){
	arr := []int{5,2,1,1,7,6,6}
	fmt.Println(b_sort(arr))

}